android/source/src/main/kotlin/com/gu/source/components/banner/AlertBanner.kt

```diff
@@ -0,0 +1,315 @@
+            )
+
+            Informative -> AppColour(
+                light = Source.Palette.Brand800,
+                dark = Source.Palette.Brand400,
+            )
+
+            Error -> AppColour(
+                light = Source.Palette.News600,
+                dark = Source.Palette.News100,
+            )
+        }
+
+        internal fun getIcon(): ImageVector? = when (this) {
+            Informative -> Style.iconInfoRound
+
+            Error -> Style.iconAlertRound
+
+            Neutral -> null
+        }
+
+        internal fun getIconTint() = when (this) {
+            Informative -> AppColour(
+                light = Source.Palette.Brand400,
+                dark = Source.Palette.Brand800,
+            )
+
+            Error -> AppColour(
+                light = Source.Palette.Error400,
+                dark = Source.Palette.Error500,
+            )
+
+            Neutral -> AppColour.Unspecified
+        }
+
+        internal fun getTextColor() = when (this) {
+            Neutral -> AppColour(
+                light = Source.Palette.Neutral100,
+                dark = Source.Palette.Neutral100,
+            )
+
+            Informative -> AppColour(
+                light = Source.Palette.Neutral0,
+                dark = Source.Palette.Neutral100,
+            )
+
+            Error -> AppColour(
+                light = Source.Palette.Neutral0,
+                dark = Source.Palette.Neutral100,
+            )
+        }
+
+        internal fun getCrossIconTint() = when (this) {
+            Neutral -> AppColour(
+                light = Source.Palette.Neutral100,
+                dark = Source.Palette.Neutral100,
+            )
+
+            Informative -> AppColour(
+                light = Source.Palette.Neutral0,
+                dark = Source.Palette.Neutral100,
+            )
+
+            Error -> AppColour(
+                light = Source.Palette.Neutral0,
+                dark = Source.Palette.Neutral100,
+            )
+        }
+    }
+}
+
+/**
+ * A composable function that displays an alert banner with optional clickable links.
+ *
+ * @param text The text to be displayed in the banner.
+ * @param priority The type of message to be displayed (Neutral, Informative, Error).
+ * @param onCloseClick A callback to be invoked when the cancel icon is clicked.
+ * @param onClick A callback to be invoked when the banner is clicked.
+ * @param modifier The modifier to be applied to the banner.
+ */
+@SuppressLint("DiscouragedApi")
+@Composable
+fun AlertBanner(
+    text: String,
+    priority: AlertBanner.Priority,
+    onCloseClick: () -> Unit,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(priority.getBackdropColour().current)
+            .padding(AlertBanner.Style.ContentPadding),
+        horizontalArrangement = Arrangement.spacedBy(AlertBanner.Style.crossTextSpacing),
```

Comment:
When you only have two elements in a collection layout, explicitly specify spacing between them. Move this as a `Spacer` before the close button.

Reply:
done